packages/core/src/core/themes/day.scss

```
@@ -1,79 +1,84 @@
+@use 'sass:list';
 @use '../tokens' as castor;
 
-@mixin day() {
-  @include castor.tokens();
+// $type?: class | raw;
+@mixin day($type1: null, $type2: null) {
+  $class: list.index(($type1, $type2), 'class');
+  $raw: list.index(($type1, $type2), 'raw');
```

[Minor] You might benefit a very little from `$types: ($type1, $type2);` 😏

```suggestion
  $types: ($type1, $type2);
  $class: list.index($types, 'class');
  $raw: list.index($types, 'raw');
```

---

Since it runs during build, not in runtime, doesn't matter much.
However, it does read better, I like that.